tdp_vars_defaults/yarn/yarn.yml

```
@@ -45,6 +45,8 @@ yarn_site:
   yarn.resourcemanager.system-metrics-publisher.enabled: "true"
   yarn.resourcemanager.recovery.enabled: "true"
   yarn.resourcemanager.store.class: "org.apache.hadoop.yarn.server.resourcemanager.recovery.ZKRMStateStore"
+  yarn.resourcemanager.zk-state-store.parent-path: "/rmstore"
+  yarn.resourcemanager.ha.automatic-failover.zk-base-path: "/yarn-leader-election"
```

```suggestion
  yarn.resourcemanager.ha.automatic-failover.zk-base-path: "/yarn-leader-election"
  hadoop.zk.acl: world:anyone:r,sasl:rm:rwcda
```

I tested a deployment while adding this property, it creates the zNodes with the proper ACLs (I had to get the jaas part from this PR for this to work)

```
[zk: master-01.tdp:2181,master-02.tdp:2181,master-03.tdp:2181(CONNECTED) 0] ls /
[hiveserver2, zookeeper, yarn-leader-election, hadoop-ha, rmstore, hadoop-kms, hbase]
[zk: master-01.tdp:2181,master-02.tdp:2181,master-03.tdp:2181(CONNECTED) 1] getAcl /yarn-leader-election
'world,'anyone
: r
'sasl,'rm
: cdrwa
[zk: master-01.tdp:2181,master-02.tdp:2181,master-03.tdp:2181(CONNECTED) 2] getAcl /rmstore             
'world,'anyone
: r
'sasl,'rm
: cdrwa
```

---

Good catch, I did not think to look in the core-site properties.

I'll refactor the PR only adding this property and the jaas for the YARN RM

---

Done in the latest push force